crates/frontend/src/contextmenu/mod.rs

```diff
@@ -24,56 +24,61 @@ impl ContextMenu {
         }
     }
 
-    // rendering context menu for folder
     pub fn folder_menu_render(
         context_menu: &ContextMenu
     ) -> Dom {
         html!("div", {
-                .apply(|dom| styles::contextmenu::body(dom, &context_menu.position.0, &context_menu.position.1))
-                .children(&mut [
-                    html!("div", {
-                        .text("New Folder")
-                        .apply(styles::contextmenu::option)
-                        .event(clone!(context_menu => move |_event: events::MouseDown| {
-                            web_sys::console::log_1(&"New Folder Created".into());
-                            context_menu.add_folder();
-                        }))
-                    }), 
-                    html!("div", {
-                        .text("New File")
-                        .apply(styles::contextmenu::option)
-                        .event(clone!(context_menu => move |_event: events::MouseDown| {
-                            web_sys::console::log_1(&"New File Created".into());
-                            context_menu.add_file();
-                        }))
-                    }),
-                    html!("div", {
-                        .text("Rename Folder")
-                        .apply(styles::contextmenu::option)
-                        .event(clone!(context_menu => move |_event: events::MouseDown| {
-                            web_sys::console::log_1(&"Renaming Folder".into());
-                            if let Target::Directory(dir) = &context_menu.target  {
-                                RENAME.with(|rename| {
-                                    rename.set(Some(Target::Directory(dir.clone())));
-                                });
-                            }
-                        }))
-                    })
-                ])
-            })
+            .class("absolute")
+            .class("z-[1000]")
+            .class("w-60")
```

Comment:
A lot of these classes are just adding a single style to the element, perhaps we just replace them with calls to `DomBuilder::style`? What do you think?

![image](https://github.com/user-attachments/assets/5bbb143d-a74f-4168-aaa5-92cd6c61b430)